docs/content/documentation/systems/Materials/tensor_mechanics/ComputeElasticityTensor.md

```diff
@@ -1,10 +1,181 @@
-<!-- MOOSE Documentation Stub: Remove this when content is added. -->
-
-# ComputeElasticityTensor
+# Compute Elasticity Tensor
 !syntax description /Materials/ComputeElasticityTensor
 
+## Description
+The material `ComputeElasticityTensor` builds the elasticity (stiffness) tensor with various user-selected material symmetry options.
+`ComputeElasticityTensor` also rotates the elasticity tensor during the initial time step only; this class does not rotate the elasticity tensor during the simulation.
```

Comment:
Maybe I'd say it optionally rotates the elasticity tensor during initialization if Euler angles are provided.

Reply:
I added a sentence below the marked diff with your note about if Euler angles are provided.